ios/Podfile

```diff
@@ -35,14 +31,25 @@ def react_native_pods()
   pod 'react-native-fetch-blob', :path => '../node_modules/react-native-fetch-blob'
   pod 'react-native-pdf', :path => '../node_modules/react-native-pdf'
 
-end 
+end
+
+def react_native_modules()
+  pod 'RNModules', path: './RNModules', subspecs: [
+    'RNLogging',
+    'RNTranslationManager',
+    'RNCurrencyManager',
+    'RNDeviceInfo'
+  ]
+end
 
 target 'RNPlayground' do
   react_native_pods()
+  react_native_modules()
 end
 
 target 'RNKiwiMobile' do
   react_native_pods()
+  react_native_modules()
 
   pod 'RNNavigator', :path => '../node_modules/@kiwicom/react-native-native-modules'
```

Comment:
We need to remove this one, we will use same approach as https://github.com/kiwicom/mobile/pull/672.